src/tests/Equinor.ProCoSys.IPO.Test.Common/ReadOnlyTestsBase.cs

```diff
@@ -91,9 +93,15 @@ protected Project GetProjectById(int projectId)
             return context.Projects.Single(x => x.Id == projectId);
         }
 
-        protected Person AddPerson(IPOContext context, Guid oid, string firstName, string lastName, string userName, string email)
+        protected Person AddPerson(IPOContext context, Guid oid, string firstName, string lastName, string userName, string email, int? id = null)
         {
             var person = new Person(oid, firstName, lastName, userName, email);
+
+            if (id != null)
```

Comment:
Undo all changes in this class. If special setup needed in GetInvitationsForExportQueryHandlerTests.cs, do it there.
.SetProtectedIdForTesting should not be used at all when setting up TestBase. Identities get real id's when saved in context.SaveChangesAsync(). I see that GetInvitationsForExportQueryHandlerTests also use SetProtectedIdForTesting, must be rewritten

Reply:
I can support on this issue